src/Symfony/Bridge/Doctrine/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 CHANGELOG
 =========
+5.2.0
+-----
+
+ * added support for symfony/uid as types
```

Comment:
I would suggest mentioning what this PR provides by giving the name of the new classes